nodejs/src/connection.rs

```diff
@@ -95,10 +122,30 @@ impl Connection {
         Ok(Table::new(tbl))
     }
 
+    #[napi]
+    pub async fn create_empty_table(
+        &self,
+        name: String,
+        schema_buf: Buffer,
+        mode: String,
+    ) -> napi::Result<Table> {
+        let schema = ipc_file_to_schema(schema_buf.to_vec())
+            .map_err(|e| napi::Error::from_reason(format!("Failed to read IPC file: {}", e)))?;
```

Comment:
I think the user would be confused by the idea of an "IPC file" involved here. This is really about the schema, right?
```suggestion
            .map_err(|e| napi::Error::from_reason(format!("Failed to parse schema: {}", e)))?;
```

Reply:
Yes, we use IPC to marshal schemas to and from Rust (we should ideally come up with something based on the C API, I'll make a ticket) Good idea.  I went with something slightly different `Failed to marshal schema from JS to Rust` just because the most likely cause is an internal bug of some kind something and not an invalid schema.